midterm/midterm_v3.py

```diff
@@ -0,0 +1,159 @@
+from myarray import myArray2D, linearSet, binarySet, SparseMatrix
+
+
+# method #2 to read entry element from txt file and store those elements in list of list
+def readfile(input_file):
+    big_lst = []
+    with open(input_file) as f:
+        lines = f.readlines()
+        for i in range(len(lines)):
+            entry_row = lines[i].split()
+            big_lst.append([int(entry) for entry in entry_row])
+    return big_lst
+
+
+data = readfile("mA.txt")
+
+# load data to matrix
+def loadMatrix(data):
+    numRow = len(data)
+    numCol = len(data[0])
+    newMatrix = myArray2D(numRow, numCol)
+    i = 0
+    for row in data:
```

Comment:
for i in numRow: 
  for j in numCol: 
     newMatrix[i, j] = data[i][j]
return newMatrix

Reply:
similar to L35-37